File: july 11 hw/1015A/code.py
n, m = map(int, input().split())

cnt = 0
log = []
ans = []
    
for _ in range(n):
    l, r = map(int, input().split())
    log.append([l,r])
    
for i in range(1,m+1):
    for j in range(n):
        if (log[j][0] <= i and i<= log[j][1]):
            break
    else:
        cnt+=1
        ans.append(i)
       
print(cnt)
for i in range(len(ans)):
    print(ans[i],end=" ")
    

            
        
    

File: july 11 hw/1015A/code.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner inp = new Scanner(System.in);
        
        int n = inp.nextInt();
        int m = inp.nextInt();
        
        int cnt = 0;
        
        ArrayList<int[]> log = new ArrayList<>();
        ArrayList<Integer> ans = new ArrayList<>();
        
        for (int i = 0; i < n; i++) {
            int l = inp.nextInt();
            int r = inp.nextInt();
            log.add(new int[]{l, r});
        }
        
        for (int i = 1; i <= m; i++) {
            boolean covered = false;
            for (int j = 0; j < n; j++) {
                int[] interval = log.get(j);
                if (interval[0] <= i && i <= interval[1]) {
                    covered = true;
                    break;
                }
            }
            if (!covered) {
                cnt++;
                ans.add(i);
            }
        }
        
        System.out.println(cnt);
        for (int i = 0; i < ans.size(); i++) {
            System.out.print(ans.get(i) + " ");
        }
    }
}
